package.json

```diff
@@ -32,21 +32,24 @@
     "version": "ALLOW_MISSING_VARIABLES=1 yarn build"
   },
   "dependencies": {
+    "@chakra-ui/react": "^2.8.2",
```

Comment:
Can we depend only on the tabs component (@chakra-ui/tabs)?

Reply:
No, we can't rely on it alone.

After updating the import from the package, we're now only consuming 5KB of it.